content/2017/05/27/liveqa-biz.rst

```diff
@@ -0,0 +1,94 @@
+public: yes
+author: sondra
+type: event
+tags: [Event, Q&A, Tips, Slack, Startup, Business Development]
+image:
+  - src: 'blog/2017/liveqa-biz/desk-mobile.jpg'
+summary: |
+  Are you a startup founder
+  looking to develop a new web app
+  or a company interested in expanding your web services?
+  Here's your chance
+  to ask all those questions
+  that keep you up at night.
+
+  .. callmacro:: content.macros.j2#link_button
+    :url: 'http://friends.oddbird.net'
+
+    Join the channel for our Q&A
+
+
+Event: Live Q&A with OddBird
+============================
+
+Date: **Thursday, June 1** |br|
+Time: **11:30am - 12:30pm** (MST) |br|
+Location: `OddFriends Slack`_
+
+.. _OddFriends Slack: http://friends.oddbird.net
+
+.. |br| raw:: html
+
+  <br/>
+
+
+Questions like:
+...............
+
+- How much does an MVP or prototype cost?
+- How do I validate my concept with real users?
+- Who should be on my development team?
+- How can I communicate effectively with developers?
+- What technology should I use?
+- How can I build a user-base?
+- What's the difference between a mockup and a wireframe?
+
+It's your hour.
+What do you want to know,
+from a web-product experts in the field?
+
+Who Are These Experts?
+......................
+
+As core devs on Django Project, Susy, & Plone —
+we're `experts`_ in tool building.
+
+.. _experts: /birds/
+
+We lead many teams, both tiny startups
+and huge enterprise companies
+through the web app development process —
+from branding and naming
+to content strategy and information architecture
+to project estimatation and MVP.
+Recently, one of our clients' companies
+was acquired!
```

Comment:
It amuses me that this is a good thing, I guess? The excitement (`!`) catches me off guard, but I trust that @SondraE knows our audience better than I do. 😄